xml_inscriptions/bib_only/MA.Bos.MFA.G.84.48.xml

```diff
@@ -141,30 +146,31 @@
    <text>
       <body>
          <div type="edition">
-            <ab><gap extent="unknown" unit="character" reason="illegible"></gap>εξοδ<gap extent="unknown" unit="character" reason="illegible"></gap> <lb/>
-               
-               <gap extent="unknown" unit="character" reason="illegible"></gap><unclear>  γ̣</unclear>αριστω Βάκχω <gap extent="unknown" unit="character" reason="illegible"></gap>  <lb/>  
-               
-               <gap extent="unknown" unit="character" reason="illegible"></gap>ααι ἄμπελοι αἱ ἐν τῷ Ῥοδι <lb/>
-               
-               <supplied reason="undefined"><gap extent="unknown" unit="character" reason="illegible"></gap>κο</supplied>σίων ἄξιαι καὶ ιγαιλ<unclear> </unclear>σεαριαδ <gap extent="unknown" unit="character" reason="illegible"></gap> <lb/>
-               
-                 <gap extent="unknown" unit="character" reason="illegible"></gap> η καὶ πι<unclear>θα</unclear>νὸν <unclear> </unclear>α<unclear> </unclear>ματισμον  <lb/>
-               
-               <gap extent="unknown" unit="character" reason="illegible"></gap> ἄξια σ<unclear>τα</unclear>τήρων ὀγδοήκοντα <lb/>
-               <gap extent="unknown" unit="character" reason="illegible"></gap> τᾶς προσόδ<supplied reason="undefined">ω</supplied> <supplied reason="undefined">τ</supplied>ᾶς <gap extent="unknown" unit="character" reason="illegible"></gap>  <lb/>
-               
-               <gap extent="unknown" unit="character" reason="illegible"></gap>λοισι τὰ δύο μέρη τῷ πατ<unclear>ρ</unclear><supplied reason="undefined">ὶ</supplied> <lb/>
-               
-               ασοισι δὲ καὶ τὸ ε<unclear> </unclear>ρτωνπα<gap extent="unknown" unit="character" reason="illegible"></gap><lb/>
-               
-               <gap extent="unknown" unit="character" reason="illegible"></gap>καταν μάτερα Μεγιστώ <gap extent="unknown" unit="character" reason="illegible"></gap></ab></div>
+            <ab>
+               <lb n="1"/><gap extent="unknown" unit="character" reason="illegible"></gap>εξοδ<gap extent="unknown" unit="character" reason="illegible"></gap> 
+               <lb n="2"/><gap extent="unknown" unit="character" reason="illegible"></gap><unclear>  γ̣</unclear>αριστω Βάκχω <gap extent="unknown" unit="character" reason="illegible"></gap>   
+               <lb n="3"/><gap extent="unknown" unit="character" reason="illegible"></gap>ααι ἄμπελοι αἱ ἐν τῷ Ῥοδι 
+               <lb n="4"/><supplied reason="undefined">κο</supplied>σίων ἄξιαι καὶ ιγαιλ<unclear> </unclear>σεαριαδ <gap extent="unknown" unit="character" reason="illegible"></gap> 
+               <lb n="5"/><gap extent="unknown" unit="character" reason="illegible"></gap> η καὶ πι<unclear>θα</unclear>νὸν <unclear> </unclear>α<unclear> </unclear>ματισμον  
```

Comment:
What you've put here as empty `<unclear>` elements I think is Merkelbach's way of saying that there is an illegible character there, or a lost character, but that he (thinks he) knows how many characters are missing. Thus I'm not convinced that `<unclear>` is correct here, but we can go back to edit.